README.md

```diff
@@ -17,21 +17,21 @@ the square brackets (for multiple-choice questions).
 
 Select all that apply:
 ```
-[] Strings
-[] Booleans
-[] Undefined
+[x] Strings
+[x] Booleans
+[x] Undefined
 [] NaN
-[] Integers
+[x] Integers
 [] Arrays
-[] Null
+[x] Null
 ```
```

Comment:
:+1: Although technically "Integer" is not a Javascript data type -- "Number" is. Not counting it wrong though!